ct/ipfs.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+source <(curl -s https://raw.githubusercontent.com/tteck/Proxmox/main/misc/build.func)
+# Copyright (c) 2021-2024 tteck
+# Author: tteck (tteckster)
+# Co-Author: ulmentflam
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+
+function header_info {
+clear
+cat <<"EOF"
+    ________  ___________
+   /  _/ __ \/ ____/ ___/
+   / // /_/ / /_   \__ \
+ _/ // ____/ __/  ___/ /
+/___/_/   /_/    /____/
+EOF
+}
+header_info
+echo -e "Loading..."
+APP="IPFS"
+var_disk="4"
+var_cpu="2"
+var_ram="6144"
```

Comment:
Really? That's a lot of RAM

Reply:
I got conflicting reports on the minimum ram requirements, I believe it should run on 4096. I can update this.